release/_publishing.sh

```diff
@@ -259,7 +270,10 @@ function upload_release {
 
 			gsutil cp "${_BUILD_DIR}/release/${file}" "gs://liferay-releases/${LIFERAY_RELEASE_PRODUCT_NAME}/release-candidates/${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}/"
 
-			scp "${file}" root@lrdcom-vm-1:"/www/releases.liferay.com/${LIFERAY_RELEASE_PRODUCT_NAME}/release-candidates/${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}"
+			if [ "${ssh_connection}" == "true" ]
```

Comment:
@kiwm, you've the same logic in 2 different methods, I mean, in 2 methods you need to know if the SSH connection was successful, am I right?

How about create a new function which provides this info and invoke it in the 2 needed places?